tutorials/preprocessing/70_fnirs_processing.py

```diff
@@ -26,6 +26,24 @@
 raw_intensity.load_data()
 
 
+###############################################################################
+# Providing more meaningful annotation information
+# ------------------------------------------------
+#
+# First, we attribute more meaningful names to the trigger codes which are
+# stored as annotations. Further, we include information about the duration of
+# each stimulus, which was 5 seconds for all conditions in this experiment.
+# Further, we remove the trigger code 15, which signaled the start and end
+# of the experiment and is not relevant to analysis.
+
+raw_intensity.annotations.set_durations(5)
+raw_intensity.annotations.rename({'1.0': 'Control',
+                                  '2.0': 'Tapping/Left',
+                                  '3.0': 'Tapping/Right'})
+raw_intensity.annotations.delete(
+    np.where([d == '15.0' for d in raw_intensity.annotations.description]))
```

Comment:
In theory (to be tested) this should "do the right thing" because NumPy should be able to do the array-of-strings comparison to yield a bool array, and `np.delete` should be bool-mask-aware
```suggestion
raw_intensity.annotations.delete(
    raw_intensity.annotations.description == '15.0')
```

Reply:
Looks like this didn't work @larsoner. I'll revert tomorrow, but now it's late.

Reply:
Sounds good, no rush. Would be good to see why it didn't work...